ui/components/component-library/text-field-base/text-field-base.stories.js

```diff
@@ -141,16 +141,6 @@ export default {
   },
   args: {
     placeholder: 'Placeholder...',
-    autoFocus: false,
-    defaultValue: '',
-    disabled: false,
-    error: false,
-    id: '',
-    readOnly: false,
-    required: false,
-    size: SIZES.MD,
-    type: 'text',
-    truncate: false,
```

Comment:
Removing default args, although I think this looks better it's in danger of becoming out of sync with the components default prop so going to remove